src/inverseRobot.js

```diff
@@ -23,8 +23,19 @@
  *
  * @return {object}
  */
+
 function inverseRobot(robot) {
-  // write code here
+  const reversedObj = {};
+
+  for (const properties in robot) {
+    for (const reversedProp in reversedObj) {
+      if (robot[properties] === reversedProp) {
+        return null;
+      }
+    } reversedObj[robot[properties]] = properties;
+  }
```

Comment:
```suggestion
  for (const properties in robot) {
    for (const reversedProp in reversedObj) {
      if (robot[properties] === reversedProp) {
        return null;
      }
    } 
    reversedObj[robot[properties]] = properties;
  }
```
move statements to separate lines